restapi/api/routes.go

```diff
@@ -17,22 +18,19 @@ func registerRoutes(router *gin.RouterGroup) {
 	device.PUT("/enable-condition", EnableDeviceCondition)
 	device.POST("/disable-condition", DisableDeviceCondition)
 
-	router.GET("/reserved-devices", reservation.GetReservedDevices)
-	reservations := router.Group("/reserve/:udid")
-	reservations.Use(DeviceMiddleware())
-	reservations.POST("/", reservation.ReserveDevice)
-	reservations.DELETE("/", reservation.ReleaseDevice)
+	device.POST("/reserve", ReserveDevice)
```

Comment:
let's call it `reservations` instead of `reserve` to be more RESTful. 
Then you can also rename /reserve-devices to just reservations, so it is more consistent.

Reply:
done